builtin/mainmenu/tab_online.lua

```diff
@@ -174,6 +174,13 @@ local function get_formspec(tabview, name, tabdata)
 				"server_view_clients.png") .. ";btn_view_clients;]"
 		end
 
+		if selected_server.url then
+			retval = retval .. "tooltip[btn_server_url;" .. fgettext("Server Website") .. "]"
```

Comment:
better like this imo
```suggestion
			retval = retval .. "tooltip[btn_server_url;" .. fgettext("Visit server website") .. "]"
```

Reply:
I disagree. Adding the "visit" makes it sound like the button would open the website directly - at least for me.

Reply:
I think button tooltips should describe the action it performs, do you have a different suggestion?

Reply:
Maybe "open"? Not a native speaker but for me that feels more like a term to describe the general action here.